### PR TITLE
Bugfix/create file path

### PR DESCRIPTION
**Pull request by justinehell**
Wrong path was used when creating a file.
Update `CreateFileEvent` by adding new `path` param
Update all functions that called this event
 - `onCreateFileEvent` function in `ModelizerTextView.vue`
 - `onSubmit` function in `CreateFileForm.vue`
 -  `createProjectFolder` function in `Project.js`

### CODE REVIEW

File: src/composables/events/FileEvent.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const SelectNodeEvent = new Subject();
 
 /**
  * Represent a rxjs Event object to emit and to receive events about file creation.
- * The event should be an Object that contains the `name` (path) of the file and `isFolder`.
+ * The event should be an Object that contains the `name`, `path` of the file and `isFolder`.
```
Comment (k-y0u) [inline src/composables/events/FileEvent.js:29]: you can remove `of the file`

File: src/components/form/CreateFileForm.vue [hunk 2]
```diff
@@ -68,23 +68,24 @@ const submitting = ref(false);
 function onSubmit() {
   const parent = props.file.id === props.projectName ? '' : `${props.file.id}/`;
   const promise = props.isFolder ? createProjectFolder : writeProjectFile;
-  const path = props.isFolder ? `${parent}${fileName.value}` : new FileInput({
+  const filePathOrFileInput = props.isFolder ? `${parent}${fileName.value}` : new FileInput({
```
Comment (k-y0u) [inline src/components/form/CreateFileForm.vue:71]: weird

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: src/components/form/CreateFileForm.vue (hunk 2 above)
Comment (justinehell) [reply src/components/form/CreateFileForm.vue]: agreed, changed

File: src/composables/events/FileEvent.js (hunk 1 above)
Comment (justinehell) [reply src/composables/events/FileEvent.js]: done

File: tests/unit/components/ModelizerTextView.spec.js [hunk 3]
```diff
@@ -437,7 +437,6 @@ describe('Test component: ModelizerTextView', () => {
 
     it('should update activeFileTab, send ExpandFolder events on Folder', async () => {
       FileEvent.OpenFileEvent.next = jest.fn();
-      wrapper.vm.activeFileTab = { isSelected: false, id: 'fileName' };
```
Comment (k-y0u) [inline tests/unit/components/ModelizerTextView.spec.js:440]: I'm not sure if this part should be remove.

**Review by k-y0u — APPROVED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Review by k-y0u — APPROVED** [review]

**Pull request merged**